Lib/test/_test_multiprocessing.py

```diff
@@ -6415,6 +6420,8 @@ def setUp(self):
         sys.path[:] = [p for p in sys.path if p]  # remove any existing ""s
         sys.path.insert(0, self._temp_dir)
         sys.path.insert(0, "")  # Replaced with an abspath in child.
+        self.assertIn(self.start_method, self.START_METHODS)
+        self._ctx = multiprocessing.get_context(self.start_method)
         try:
             self._ctx_forkserver = multiprocessing.get_context("forkserver")
```

Comment:
Is it needed?

Reply:
hehe, thanks! I meant to delete the old ctx creations.